extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -69,6 +69,23 @@ export let InteractiveComponentDef;
  */
 export let InteractiveReactData;
 
+/**
+ * @typedef {{
+ *   brand-favicon: ?string,
+ *   brand-label: string,
+ *   cta-text: ?number,
+ *   product-details: string,
+ *   product-images: string,
+ *   product-price: string,
+ *   product-tag-id: string,
+ *   product-title: string,
+ *   reviews-data: ?string,
```

Comment:
Will this be a string?

Reply:
Originally this was going to be a URL to review data so it could build the rating dynamically.
Since we want this component to render without making a request, I think this needs to be static. 

This could be called `aggregate-rating` to follow the naming convention in the [product schema](https://developers.google.com/search/docs/advanced/structured-data/product).

It could be something like:
```
"aggregate-rating": {
  "ratingValue": "4.4",
  "reviewCount": "89",
  "reviews-url": "url to site to read reviews"
}
```

Reply:
It might be easier to add pieces to config as we code them. 

This PR could contain just `product-tag-id` and `product-title` since they are the ones pertinent to this PR.
It's up to you but I think it will help the PR get merged faster if we go this route.

Reply:
Ok, sounds good. I think it's easier if I just keep what I have for now. We can modify the data and tests ad-hoc when we are implementing the other features.